test/utils/setupBabel.js

```diff
@@ -0,0 +1,3 @@
+require('@babel/register')({
+  extensions: ['.js', '.ts'],
```

Comment:
> possibly setup infra for tests written in TypeScript

I believe it's already a great step.

Should we include the `.tsx` extension, in anticipation of .test.tsx files? For instance, https://github.com/mui-org/material-ui-x/blob/master/packages/grid/x-grid/test/XGrid.test.tsx has TypeScript linting issues but runs.

Reply:
One step at a time 😉